cabal.project

```diff
@@ -77,3 +69,35 @@ allow-newer: servant-multipart-api:servant
 
 -- ghc 9
 allow-newer: tdigest:base
+
+-- ghc 9.2
+allow-newer: basement:base
+allow-newer: binary:bytestring
+allow-newer: generics-sop:base
+allow-newer: generics-sop:ghc-prim
+allow-newer: generics-sop:template-haskell
+allow-newer: http-media:base
+allow-newer: lucid:base
+allow-newer: memory:base
+allow-newer: openssl-streams:bytestring
+allow-newer: servant-js:base
+allow-newer: servant-js:base-compat
+allow-newer: servant-js:lens
+allow-newer: servant-multipart:bytestring
+allow-newer: servant-multipart:bytestring
+allow-newer: servant-multipart:http-media
+allow-newer: servant-multipart:lens
+allow-newer: servant-multipart:resourcet
+allow-newer: servant-swagger:base
+allow-newer: servant-swagger:base-compat
+allow-newer: servant-swagger:base-compat
+allow-newer: servant-swagger:Cabal
+allow-newer: singleton-bool:base
+allow-newer: sop-core:base
+allow-newer: swagger:base
```

Comment:
Why was this added? The bounds are `>=4.6 && <5.0` which is wide enough.

Reply:
🤷 it was needed when I opened this PR 3 months ago